pkgs/by-name/go/gofumpt/package.nix

```
@@ -24,31 +23,27 @@ buildGoModule rec {
 
   ldflags = [
     "-s"
-    "-X main.version=v${version}"
+    "-X main.version=v${finalAttrs.version}"
   ];
 
+  nativeCheckInputs = [ versionCheckHook ];
```

seems like version check should be in install check https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook
```suggestion
  doInstallCheck = true;
  nativeInstallCheckInputs = [ versionCheckHook ];
```